cfme/control/explorer/alert_profiles.py

```diff
@@ -205,7 +205,10 @@ def assign_to(self, assign, selections=None, tag_category=None):
 @attr.s
 class AlertProfileCollection(BaseCollection):
 
-    ENTITY = BaseAlertProfile
+    def instantiate(self, *args, **kwargs):
+        alert_profile_class = args[0]
```

Comment:
@psav i think we need to reserve some time to collect the loose end about collections with polymorphic elements